tests/dummy/public/test-forms/basic-fields/data.ttl

```diff
@@ -19,6 +22,13 @@
 # Input
 <http://ember-submission-form-fields/source-node> <http://mu.semte.ch/vocabularies/ext/inputValue> "Simple value".
 
+# Currency Input
+esff:source-node ext:currencyInputValue eurio:currencyInput.
+
+eurio:currencyInput a eurio:MonetaryAmount ;
+    eurio:value "10000.00"^^xsd:decimal ;
+    eurio:currency "EUR" .
+
```

Comment:
The table in the dummy app that shows the triples:

![image](https://github.com/lblod/ember-submission-form-fields/assets/36266973/64249678-32ee-4b09-a8f7-894871fd7224)
![image](https://github.com/lblod/ember-submission-form-fields/assets/36266973/9c284e4d-91ae-4fcf-906c-7d86fedf2bd2)

Reply:
The value is typed as an integer as the fractions are 00